tests_e2e/tests/agent_update/rsm_update.py

```diff
@@ -49,16 +50,17 @@ def __init__(self, context: AgentTestContext):
             ip_address=self._context.vm_ip_address,
             username=self._context.username,
             private_key_file=self._context.private_key_file)
+        self._installed_agent_version = "9.9.9.9"
 
     def get_ignore_error_rules(self) -> List[Dict[str, Any]]:
         ignore_rules = [
             #
             # This is expected as we validate the downgrade scenario
             #
             # WARNING ExtHandler ExtHandler Agent WALinuxAgent-9.9.9.9 is permanently blacklisted
-            #
+            # Note: Version varies depending on the pipeline branch the test is running on
             {
-                'message': r"Agent WALinuxAgent-9.9.9.9 is permanently blacklisted"
+                'message': rf"Agent WALinuxAgent-{self._installed_agent_version} is permanently blacklisted"
```

Comment:
why using the version of the installed agent? shouldn't it be the version that your test uses for the downgrade?

Reply:
no, we blocklist the current version which is running not the downloaded version